index.html

```diff
@@ -5,16 +5,17 @@
     <title>Mini Weather App</title>
     <link rel="stylesheet" href="css/style.css">
```

Comment:
Make sure your custom CSS file is loaded after any libraries you're using (i.e. bootstrap and flags).